{
	"catchErrorName": {
		"prefix": "catchErrorName",
		"body": "catch(error) {${2|console.log,alert|}(`${error.name}\\n${error.stack}`)}",
		"description": "It's useful to catch error in JS"
	}
}